variables.tf

```diff
@@ -1013,6 +1009,11 @@ variable "rbac_aad_tenant_id" {
   default     = null
 }
 
+variable "resource_group_name" {
```

Comment:
`resource_group_name` is still a required variable so we should move it back to the original place.